server/gateway_test.go

```diff
@@ -7349,3 +7349,63 @@ func TestOCSPGatewayMissingPeerStapleIssue(t *testing.T) {
 	waitForOutboundGateways(t, srvC, 2, 5*time.Second)
 	wg.Wait()
 }
+
+func TestGatewayOutboundDetectsStaleConnectionIfNoInfo(t *testing.T) {
```

Comment:
we've had some reports of idle websocket connections not pruned by the ping interval but peer actually disconnected already, so I wonder this could be a similar symptom or related for those type of connections

Reply:
I think it might be different issue then, just that in websocket connections it is possible for a client connection to look connected and ping interval not happening so it would still show in connz but tcp connection is closed already.